BDArmory/Distribution/GameData/BDArmory/ChangeLog.txt

```diff
@@ -20,6 +20,12 @@
 			- IMPORTANT: "LoftTermRange" has been renamed to "terminalHomingRange"!
 		- Fixed MultiMissileLauncher behavior with loft guidance, as well as pronav/augpronav.
 		- Added new "aamhybrid" homingType, uses lead guidance until "terminalHomingRange" is reached, then it will switch to "terminalHomingType", with the same options as loft guidance.
+        - new 'hasDataLink' field that can be used to Launch Radar/Heat missiles and guide them later with help of an external radar or out of range of your radar/seeker an re-acquire a lock when inside radar/seeker range.
+		    - Datalink Can also guide missiles outside they Max offboresight, but with a precision lost until the target get inside the boresight again
+		- new 'DataLinkDrift' field that defines the inertial drift of the target when it's on guided via Datalink when it's outside missile max offboresight or without a lock if radarTimeout > 15
+		- new 'hasIFF' field (default:true) for Heatseekers that define if the missile can indentify what target is friendly or not, if it don't have it just home in the bigger heat signature be it friendly or not
```

Comment:
You haven't changed any of the DUMB_IR_SEEKER code, so this isn't true. Heaters will continue to heat towards the heatSig that closest matches the original target.